docs/application/web/guides/device/system-information.md

```diff
@@ -234,6 +234,31 @@ To receive notifications on property value changes:
    var id = tizen.systeminfo.addPropertyValueArrayChangeListener('SIM', successCallback);
    ```
 
+
+<b>Remark:</b>
+
+In case of `WIFI_NETWORK` property, value change listener is triggered on `ipAddress` and `ip6Address` properties change (the network layer). These changes could be not consistent with physical layer (`status` or `signalStrength` properties of physical adapter).
+
+According to above constraints, in specific situation, the listener could be triggered just before network adapter shutdown and the value of `status` returned by listener would be outdated.
```

Comment:
Please advice me here, the following is my understanding:

According to the previous constraints, in a specific situation, the listener could be triggered just before the network adapter shutdown. In this case, the value of `status` returned by the listener will be outdated. 

Has the rephrase alter the original meaning of the text?